Unity-Technologies/ScriptableRenderPipeline
# PR for merging Urp/doc/2d/image update into Master.

**DONT FORGET TO ADD A CHANGELOG**

### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-2019.3` label. After you backport the PR, the label changes to `backported-2019.3`.
- [ ^] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [ ^] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
To merge this branch that contains updates to the 2D Renderer overview and setup pages.
---
### Testing status

**Manual Tests**: What did you do?
- Other: 
Checked formatted pages as they would appear online.
**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.
